tests/smoke-cy.js

```diff
@@ -50,7 +50,7 @@ describe("DC/OS UI [00j]", function() {
     });
 
     it("displays one row on the table [00t]", function() {
-      cy.get("table tbody tr").should("to.have.length", 3).contains("dcos-01");
+      cy.get("table tbody tr").should("to.have.length", 6).contains("dcos-01");
```

Comment:
Why does this change the number of rows?

Reply:
Good question. In fact even without my change there were more than 3 of them. The difference I observe that without my change `/dcos-history-service/` call would error out in Cypress, with the change it doesn't. Looks like it is something that can make or break our test.

Reply:
makes sense